internal/services/mobilenetwork/mobile_network_attached_data_network_data_source_test.go

```diff
@@ -12,6 +12,9 @@ type MobileNetworkAttachedDataNetworkDataSource struct{}
 
 func TestAccMobileNetworkAttachedDataNetworkDataSource_basic(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_mobile_network_attached_data_network", "test")
+	// Limited regional availability for Mobile Network
+	data.Locations.Primary = "eastus"
```

Comment:
Can we add an override in `.teamcity/components/settings.kt` for this instead of coding it into all the tests?

Reply:
Sure, I did not know we can do that before..

Reply:
I have no idea why the CI is failing... Do you mind take a look?